lore/end_users/status/README.md

```diff
@@ -40,3 +46,7 @@ The Zuul Pipelines counter provides an easy look at how many pipelines exist in
 ### Service Status
 
 Nodepool, Zookeeper, and Zuul are three core services that are essential to the stability of a BonnyCI cluster. If anything except the green "OK" message is displayed for these service statuses, then BonnyCI could be experiencing a full service outage.
+
+## Build Status
+
+The [Build Status page](https://travis-ci.org/BonnyCI) links to the Travis-CI page with our repositories. Yes we see the humor in a CI tool using a different CI tool for testing. This is a temporary home for us.
```

Comment:
@mlangbehn Correct. Zuul/BonnyCI does not provide anything like a historical view of build success/failures.

I'm somewhat dubious of their value as well, given that the base idea being these are contributors throwing changes at the wall. The quality of the change is completely unknown, with project members being completely unable to affect that incoming quality, and with the CI system not being able to impact that quality either. All it can do is pass judgement. What would it mean that there are a lot of failures? Is that a bad thing? Or is it a good thing, that it's preventing that much bad code from getting in?